[2025-09-04T12:30:06.804+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-04T12:30:06.812+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_scanner.scan_and_generate_dags scheduled__2025-09-04T12:00:00+00:00 [queued]>
[2025-09-04T12:30:06.815+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_scanner.scan_and_generate_dags scheduled__2025-09-04T12:00:00+00:00 [queued]>
[2025-09-04T12:30:06.815+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-04T12:30:06.819+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): scan_and_generate_dags> on 2025-09-04 12:00:00+00:00
[2025-09-04T12:30:06.823+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_scanner', 'scan_and_generate_dags', 'scheduled__2025-09-04T12:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/created_dags.py', '--cfg-path', '/tmp/tmp78zuws7h']
[2025-09-04T12:30:06.823+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=215) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-04T12:30:06.825+0000] {standard_task_runner.py:91} INFO - Job 75: Subtask scan_and_generate_dags
[2025-09-04T12:30:06.825+0000] {standard_task_runner.py:63} INFO - Started process 216 to run task
[2025-09-04T12:30:06.849+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_scanner.scan_and_generate_dags scheduled__2025-09-04T12:00:00+00:00 [running]> on host ade5f5069171
[2025-09-04T12:30:06.875+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='admin@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_scanner' AIRFLOW_CTX_TASK_ID='scan_and_generate_dags' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T12:00:00+00:00'
[2025-09-04T12:30:06.875+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-04T12:30:06.881+0000] {created_dags.py:317} INFO - Starting DAG scanner execution
[2025-09-04T12:30:06.881+0000] {created_dags.py:86} INFO - Jinja2 environment initialized successfully
[2025-09-04T12:30:06.881+0000] {created_dags.py:118} INFO - Scanning for scripts in: /opt/***/scripts
[2025-09-04T12:30:06.883+0000] {created_dags.py:123} INFO - Found 2 script files
[2025-09-04T12:30:06.883+0000] {created_dags.py:333} INFO - Found 2 script(s) to process
[2025-09-04T12:30:06.883+0000] {created_dags.py:148} WARNING - No JSON parameters found for: do_something.py
[2025-09-04T12:30:06.883+0000] {created_dags.py:350} WARNING - Skipping do_something.py - no JSON parameters
[2025-09-04T12:30:06.884+0000] {created_dags.py:148} WARNING - No JSON parameters found for: select_data.sql
[2025-09-04T12:30:06.884+0000] {created_dags.py:350} WARNING - Skipping select_data.sql - no JSON parameters
[2025-09-04T12:30:06.884+0000] {created_dags.py:358} INFO - Processing completed. Success: 0/2
[2025-09-04T12:30:06.884+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-04T12:30:06.884+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-04T12:30:06.888+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_scanner, task_id=scan_and_generate_dags, run_id=scheduled__2025-09-04T12:00:00+00:00, execution_date=20250904T120000, start_date=20250904T123006, end_date=20250904T123006
[2025-09-04T12:30:06.925+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-04T12:30:06.933+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-04T12:30:06.934+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
